### PR TITLE
Removes unnecessary params from EnhancedPaymentDetails

### PR DESCRIPTION
**Pull request by RafalOzogJudopay**
Testing:
Simulator Screenshot - iPhone 15 Pro - 2024-09-12 at 18 15 24

### CODE REVIEW

**Comment by stefan-tudor** [comment]
You have a few warnings reported around one unused variable; please address it https://sonarcloud.io/project/issues?id=Judopay_JudoKit-iOS&pullRequest=266&issueStatuses=OPEN,CONFIRMED&sinceLeakPeriod=true

**Comment by RafalOzogJudopay** [comment]
> You have a few warnings reported around one unused variable; please address it https://sonarcloud.io/project/issues?id=Judopay_JudoKit-iOS&pullRequest=266&issueStatuses=OPEN,CONFIRMED&sinceLeakPeriod=true

Thanks @stefan-tudor , adjusted

File: Source/Models/ConsumerDevice/JPConsumerDevice.h [hunk 1]
```diff
@@ -30,20 +30,26 @@
...
 
+- (nonnull instancetype)initWithThreeDSecure:(nonnull JPThreeDSecure *)threeDSecure;
+
+/**
+ * DEPRECATED
+ */
 + (nonnull instancetype)deviceWithIpAddress:(nonnull NSString *)ipAddress
                               clientDetails:(nonnull JPClientDetails *)clientDetails
                                 geoLocation:(nonnull CLLocation *)geoLocation
                                threeDSecure:(nonnull JPThreeDSecure *)threeDSecure;
```
Comment (stefan-tudor) [inline Source/Models/ConsumerDevice/JPConsumerDevice.h:52]: Why not remove this altogether? you already changed the API by removing class properties

Comment (RafalOzogJudopay) [reply Source/Models/ConsumerDevice/JPConsumerDevice.h:52]: (replied on Slack)

Comment (RafalOzogJudopay) [reply Source/Models/ConsumerDevice/JPConsumerDevice.h]: Removed

**Review by stefan-tudor — APPROVED** [review]

**Pull request merged**